src/parser/mod.rs

```diff
@@ -7979,6 +7979,27 @@ impl<'a> Parser<'a> {
         }
     }
 
+    pub fn parse_enum_values(&mut self) -> Result<Vec<EnumValue>, ParserError> {
+        self.expect_token(&Token::LParen)?;
+        let values = self.parse_comma_separated(Parser::parse_enum_value)?;
+        self.expect_token(&Token::RParen)?;
+        Ok(values)
+    }
+
+    pub fn parse_enum_value(&mut self) -> Result<EnumValue, ParserError> {
+        let str = self.parse_literal_string()?;
+        let value = match self.peek_token().token {
+            Token::Eq => {
+                // Consume the `=` token
+                self.next_token();
+                let value = self.parse_number_value()?;
```

Comment:
it seems from the docs the value can be any integer whereas `parse_number_value()` doesn't support negative/signed integers, maybe we'd want to use `parse_number()` here instead?

Reply:
Has been Changed to `parse_number`. I initially think the enum value won't be a negative value.